great_expectations/optional_imports.py

```diff
@@ -77,11 +77,13 @@ def is_version_greater_or_equal(
 )
 try:
     import sqlalchemy
+    from sqlalchemy.exc import OperationalError, SQLAlchemyError
 
     sqlalchemy_version_check(sqlalchemy.__version__)
 except ImportError:
     sqlalchemy = SQLALCHEMY_NOT_IMPORTED
-
+    SQLAlchemyError = SQLALCHEMY_NOT_IMPORTED
+    OperationalError = SQLALCHEMY_NOT_IMPORTED
```

Comment:
This is a trial balloon, not sure if this is a pattern we want to continue with.

Reply:
`OperationalError` is so general that I'm hesitant to add that here without any namespacing.